TouchSky/TouchSky/TouchSky/Views/CompletedButton.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CompletedButton.swift
+//  TouchSky
+//
+//  Created by Alexa Nicole Hugenberg on 10/26/22.
+//
+
+import SwiftUI
+
+struct CompletedButton: View {
```

Comment:
Great little abstraction for the button! This is nice, and I bet you could even consolidate this with your `ToDoButton`, but I don't necessarily think you have to.